packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -93,9 +93,32 @@ ZIP_PREFIX=instana-nodejs-layer
 ZIP_NAME=$ZIP_PREFIX.zip
 TMP_ZIP_DIR=tmp
 
-echo Will build Lambda layer with name \"$LAYER_NAME\".
+if [[ -z $REGION ]]; then
+  REGIONS=$'ap-northeast-1\nap-northeast-2\nap-south-1\nap-southeast-1\nap-southeast-2\nca-central-1\neu-central-1\neu-north-1\neu-west-1\neu-west-2\neu-west-3\nsa-east-1\nus-east-1\nus-east-2\nus-west-1\nus-west-2'
+else
+  REGIONS=$REGION
+fi
```

Comment:
```suggestion
if [[ -z $REGION ]]; then
  REGIONS=$'ap-northeast-1\nap-northeast-2\nap-south-1\nap-southeast-1\nap-southeast-2\nca-central-1\neu-central-1\neu-north-1\neu-west-1\neu-west-2\neu-west-3\nsa-east-1\nus-east-1\nus-east-2\nus-west-1\nus-west-2'
fi
```

Nice addition! I think the else branch is a no-op though and can be omitted.